.github/workflows/publish.yml

```diff
@@ -21,12 +21,12 @@ jobs:
       - name: Build instant-meilisearch
         run: yarn build
       - name: Publish with latest tag
-        if: "!github.event.release.prerelease && !contains(github.ref, 'beta')"
+        if: '!github.event.release.prerelease'
```

Comment:
In the step `Check tag format` we ensure that, in the case we are in a latest release and not a pre-release, the version of the package is strictly formatted as `vX.X.X` and not for example `vX.X.X-my-feature.0`.
With that check, we do not need the second condition anymore.